app/models/material.rb

```diff
@@ -20,14 +16,10 @@ class Material < ApplicationRecord
   validates :function, inclusion: { in: FUNCTIONS }
   validates :group, inclusion: { in: GROUPS }, allow_nil: true
   validates :ul_94, inclusion: { in: UL }, allow_nil: true
-  validates :resin, inclusion: { in: RESIN }, allow_nil: true
-  validates :flame_retardant, inclusion: { in: RETARDANT }, allow_nil: true
-  validates :reinforcement, inclusion: { in: REINFORCEMENT }, allow_nil: true
   validates :foil_roughness, inclusion: { in: FOIL_ROUGHNESS }, allow_nil: true
   validates :ipc_sm_840_class, inclusion: { in: IPC_840 }, allow_nil: true
   validates :finish, inclusion: { in: FINISH }, allow_nil: true
   validates :link, http_uri: true, if: -> { link.present? }
-  validate :filler_values
```

Comment:
Be good to remove the associated method at the end of this file as well.

Reply:
Added to this merge